operator/src/main/java/oracle/kubernetes/operator/work/NextAction.java

```diff
@@ -102,33 +87,12 @@ public Step getNext() {
     return next;
   }
 
-  /**
-   * Sets the next step.
-   *
-   * @param next Next step
-   */
-  public void setNext(Step next) {
-    this.next = next;
-  }
-
-  /**
-   * Returns the last Packet.
-   *
-   * @return Packet
-   */
-  public Packet getPacket() {
-    return packet;
-  }
-
   /** Dumps the contents to assist debugging. */
   @Override
   public String toString() {
-    StringBuilder buf = new StringBuilder();
-    buf.append('[');
-    buf.append("kind=").append(kind).append(',');
-    buf.append("next=").append(next).append(',');
-    buf.append("packet=").append(packet != null ? packet.toString() : null).append(']');
-    return buf.toString();
+    String buf = '[' +  "kind=" + kind + ',' + "next=" + next + ','
+        + "packet=" + (packet != null ? packet.toString() : null) + ']';
```

Comment:
Minor nit - I'd personally prefer multi-line formatting to make the syntax visually clear and easier to edit:
```
String buf = '[' 
   + "kind=" + kind 
   + ",next=" + next 
   + ",packet=" + (packet != null ? packet.toString() : null) 
   + ']';
```

Reply:
Okay, I'll make this change too.

Reply:
Also, shouldn't need the ternary since string expressions handle null objects:
```
String buf = '[' 
   + "kind=" + kind 
   + ",next=" + next 
   + ",packet=" + packet
   + ']';
```